src/main/java/io/vertx/core/json/JsonObject.java

```diff
@@ -1275,7 +1272,7 @@ public Map.Entry<String, Object> next() {
       final Object wrapped = wrapJsonValue(val);
 
       if (val != wrapped) {
-        return new Entry(entry.getKey(), wrapped);
+        return Map.entry(entry.getKey(), wrapped);
```

Comment:
Can you please add a comment in the code about why it's safe to use the JDK (i.e. `null` case handled above)